packages/web-components/fast-components/src/menu-item/fixtures/menu-item.html

```diff
@@ -40,8 +40,18 @@ <h2>menuitemcheckbox</h2>
     <h3>checked</h3>
     <fast-menu-item checked="true" role="menuitemcheckbox">Menu item 1</fast-menu-item>
 
-    <h2>menuitemradio</h2>
-    <fast-menu-item role="menuitemradio">Menu item 1</fast-menu-item>
+    <h4>nested sub-menu</h4>
+    <fast-menu-item style="max-width: 240px;" aria-haspopup="true">
+        Menu item 1
+        <fast-menu-item slot="nested-menu">Nested Menu item 1</fast-menu-item>
```

Comment:
Does `MenuItem` support this syntax _and_ the following for nested menu?
```html
 <fast-menu>
                <fast-menu-item>Nested Menu item 1.1</fast-menu-item>
                <fast-menu-item>Nested Menu item 1.2</fast-menu-item>
                <fast-menu-item>Nested Menu item 1.3</fast-menu-item>
            </fast-menu>
```

Reply:
No, the direct nesting of menu items without a child menu is a bug in the example, fixing now.  The child submenu is required.  Fixing now.